eav/models.py

```diff
@@ -266,6 +270,7 @@ def get_validators(self):
             'object':      validate_object,
             'enum':        validate_enum,
             'enum_multi':  validate_enum_multi,
+            'json':        value_json,
```

Comment:
This is validators dict, the name should be validate_json?

Reply:
Yeah it is a typo.